src/components/Sheet/AudacityManager.tsx

```diff
@@ -149,6 +155,9 @@ function AudacityManager(props: IProps) {
             remoteId('mediafile', mediaId, memory.keyMap as RecordKeyMap) ?? '',
           cancelled: () => false,
         })) ?? '';
+      if (mediaName === ts.expiredToken) {
+        showMessage(ts.expiredToken);
+      }
```

Comment:
line 163 has a return statement at the end of the if. Do we want a return after we show the message?